### DISCUSSION TITLE
CMakelist.txt

### DISCUSSION THREAD

**Discussion by mdnurulabsur** [openMVG/openMVG#2275] [General]
edge@edge-Precision-3640-Tower:~/openMVG/build$ cd /home/edge
rm -rf openMVG
git clone --recursive https://github.com/openMVG/openMVG.git
cd openMVG
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=RELEASE ..
make
Cloning into 'openMVG'...
remote: Enumerating objects: 34797, done.
remote: Counting objects: 100% (7144/7144), done.
remote: Compressing objects: 100% (2234/2234), done.
remote: Total 34797 (delta 5182), reused 6656 (delta 4880), pack-reused 27653
Receiving objects: 100% (34797/34797), 30.65 MiB | 19.16 MiB/s, done.
Resolving deltas: 100% (25111/25111), done.
Submodule 'src/dependencies/cereal' (https://github.com/openMVG-thirdparty/cereal.git) registered for path 'src/dependencies/cereal'
Submodule 'src/dependencies/glfw' (https://github.com/elmindreda/glfw.git) registered for path 'src/dependencies/glfw'
Submodule 'src/dependencies/osi_clp' (https://github.com/openMVG-thirdparty/osi_clp.git) registered for path 'src/dependencies/osi_clp'
Cloning into '/home/edge/openMVG/src/dependencies/cereal'...
remote: Enumerating objects: 6563, done.        
remote: Total 6563 (delta 0), reused 0 (delta 0), pack-reused 6563        
Receiving objects: 100% (6563/6563), 2.91 MiB | 12.73 MiB/s, done.
Resolving deltas: 100% (3818/3818), done.
Cloning into '/home/edge/openMVG/src/dependencies/glfw'...
remote: Enumerating objects: 28360, done.        
remote: Counting objects: 100% (204/204), done.        
remote: Compressing objects: 100% (127/127), done.        
remote: Total 28360 (delta 112), reused 129 (delta 72), pack-reused 28156        
Receiving objects: 100% (28360/28360), 14.53 MiB | 30.17 MiB/s, done.
Resolving deltas: 100% (20049/20049), done.
Cloning into '/home/edge/openMVG/src/dependencies/osi_clp'...
remote: Enumerating objects: 976, done.        
remote: Total 976 (delta 0), reused 0 (delta 0), pack-reused 976        
Receiving objects: 100% (976/976), 3.00 MiB | 29.82 MiB/s, done.
Resolving deltas: 100% (565/565), done.
Submodule path 'src/dependencies/cereal': checked out 'ac168fe78ac499be0a745bf4a3253a9660572f8d'
Submodule path 'src/dependencies/glfw': checked out 'eab31f228fd1872753c55fb438e3d88e51c0e9b2'
Submodule path 'src/dependencies/osi_clp': checked out 'a25a980c1af50cbd8962fe9d21035afa03653270'
CMake Error: The source directory "/home/edge/openMVG" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
make: *** No targets specified and no makefile found.  Stop.

Seems like the repository lacks this file. Help me with this.

**Comment by pmoulon** [comment]
Just point cmake to the path where the main CMakeLists.txt is -> in this case openMVG/src

See "configure and build" in https://github.com/openMVG/openMVG/blob/develop/BUILD.md
